modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -154,6 +260,9 @@ private void _testPostExportTask(
 	@Inject
 	private CompanyLocalService _companyLocalService;
 
+	@Inject
+	private JSONFactory _jsonFactory;
```

Comment:
Is this necessary? I don't see you are using it

Reply:
No, it isn't. I removed the code that used it and I forgot it's delition. I'm removing it currently.